k8s/operators/src/diskpool/crd.rs

```diff
@@ -19,6 +19,7 @@ derive = "Default",
 shortname = "dsp",
 printcolumn = r#"{ "name":"node", "type":"string", "description":"node the pool is on", "jsonPath":".spec.node"}"#,
 printcolumn = r#"{ "name":"status", "type":"string", "description":"pool status", "jsonPath":".status.state"}"#,
+printcolumn = r#"{ "name":"poolstatus", "type":"string", "description":"control plane pool status", "jsonPath":".status.status"}"#,
```

Comment:
what should the control-plane status be named as? @blaisedias @avishnu @Abhinandan-Purkait

Reply:
This sounds a bit confusing, why should a user be interested in the CR status? Can we not display only the control-plane status by default i.e for `kubectl get dsp -n mayastor` and show the CR status if the user does a `kubectl get dsp -owide -n mayastor`.

Reply:
I can see that the new CR status is named as `cr_state` and what I assume is control-plane pool state is `state` I think to disambiguate it should not be named `state`
Maybe `pool_state`  or  pedantically `control_plane_pool_state`.